.lintstagedrc.json

```
@@ -0,0 +1,4 @@
+{
+  "*": ["prettier --write --ignore-path .prettierignore"],
+  "**/*.{ts,js}": ["eslint --fix"]
```

```suggestion
  "**/*.{ts,js}": ["eslint"]
```


please let's not fix linting errors with githooks.
The cons I see are:
- risk of unexpected changes (unlike formatting, linting does impact how the code works)
- missing learning opportunities
- loosing control

Automatically writing format changes with prettier is fine because these are really just cosmetic changes.

---

That makes sense :ok_hand: